src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataServiceTest.java

```diff
@@ -121,7 +123,7 @@ public void before() {
             anyString()
         ))
             .thenReturn(StartEventResponse.builder()
-                .caseDetails(CaseDetails.builder().build())
+                .caseDetails(CaseDetails.builder().data(Maps.newHashMap()).build())
```

Comment:
Suggestion : use Collections.<String,Object>emptyMap() instead

Reply:
it become immutable, and we need adding something